project/Build.scala

```diff
@@ -364,7 +370,8 @@ object Build extends sbt.Build {
           "org.mockito" % "mockito-core" % mockitoVersion % "test",
           "io.spray" %%  "spray-can"       % sprayVersion,
           "io.spray" %%  "spray-routing-shapeless2"   % sprayVersion,
-          "io.spray" %%  "spray-json"    % sprayJsonVersion
+          "io.spray" %%  "spray-json"    % sprayJsonVersion,
```

Comment:
It's possible we do not need spray.json though I have not explored where it's used currently.